include/windows/osd.h

```diff
@@ -261,6 +261,7 @@ do						\
 #define strcasecmp _stricmp
 #define snprintf _snprintf
 #define sleep(x) Sleep(x * 1000)
+#define strtok_r strtok_s
```

Comment:
github doesn't make it easy to tell where a patch falls in the commit stack, but please ensure that this patch comes before strtok_r is used in the code, so we don't break the build in the middle.

Reply:
Thanks! I checked the patch order and strtok_s is used before it's effectively used in the series.